### PR TITLE
PP-1948 Feature flag allowing the deployment of the new capture mechanism

### PR DESCRIPTION
**Pull request by ckalista**
## WHAT
_A brief description of the pull request:_

- Feature flag allowing the deployment of the new capture mechanism (PP-1775) to production in a better controlled way. The feature flag is turned off by default which means capture remains synchronous until the environment variable `ASYNCHRONOUS_CAPTURE` is set to `true`.

### CODE REVIEW

**Comment by ghost** [comment]
There were errors, for info, please see...

**Comment by ghost** [comment]
There were no errors, go have a cup of coffee...

File: src/main/resources/config/config.yaml [hunk 1]
```diff
@@ -54,6 +54,8 @@ executorServiceConfig:
   timeoutInSeconds: ${AUTH_READ_TIMEOUT_SECONDS:-1}
   threadsPerCpu: ${NUMBER_OF_THREADS_PER_CPU:-100}
 
+asynchronousCapture: ${ASYNCHRONOUS_CAPTURE:-false}
```
Comment (daniele-occhipinti) [inline src/main/resources/config/config.yaml:57]: Is this the line that guarantee us the async capture will be off by default?

Comment (ckalista) [reply src/main/resources/config/config.yaml:57]: Yes, if we don't specify a `ASYNCHRONOUS_CAPTURE` environment variable the default is `false` which means capture operations are carried out in a synchronous fashion.

**Review by daniele-occhipinti — APPROVED** [review]

**Pull request merged**